Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update && \
   rm -rf /var/lib/apt/lists/* && \
   python -m pip install -U pip && \
   pip install -U setuptools wheel && \
-  pip install -U -e .[server]
+  PIP_INDEX_URL=https://gitlab.sintef.no/api/v4/projects/17883/packages/pypi/simple pip install -U -e .[server]
```

Comment:
Ouch, could this be avoided?

Reply:
Not as it is.
An alternative would be to set it as a general environment variable.

But in the end, all Docker-related files here are only used for testing purposes at this point. The actual service will be running in DataSpaces-Services (or really in DataSpaces), where it will be imported as a library instead of deployed using any of these Docker files.
So while ugly, it should not see the light of day (production-wise).